gnat2goto/driver/tree_walk.adb

```diff
@@ -456,10 +443,7 @@ package body Tree_Walk is
                                              Source_Location => Source_Loc,
                         I_Type          => Make_Pointer_Type (Make_Void_Type));
    begin
-      Append_Argument (Malloc_Args,
-                       Make_Op_Typecast (Op0             => Size,
-                                         Source_Location => Source_Loc,
-                    I_Type          => CProver_Size_T));
+      Append_Argument (Malloc_Args, Size);
```

Comment:
I'm assuming this is because we're assing `CProver_Size_T` everywhere now?

Reply:
Plus we know that `Compute_Memory_Op_Size` returns `CProver_Size_T` (I've added a check for that).